---
title: "Publications"
description: |
  We turn our findings into scientific publications.
bibliography: publications.bib
csl: apa.csl
editor_options: 
  chunk_output_type: console
output:
  distill::distill_article:
    toc: true
    toc_depth: 3
---

<script src="js/for_toc.js"></script>


```{r, echo=FALSE}
#RefManageR::BibOptions(style = "text") ## no effect :-(
bib <- RefManageR::ReadBib("publications.bib")

extract_tags <- function(bib) {
  tags <- bib$keywords
  tags <- lapply(tags, \(list) stringr::str_remove_all(string = list, pattern = "dzg\\\\_|\\\\"))
  tags <- lapply(tags, \(list) stringr::str_split(string = list, pattern = ", ", simplify = FALSE))
  table <- tidyr::unnest_longer(tibble::tibble(name = names(tags), tag = sapply(tags, \(x) x[1][[1]])), col = "tag")
  table
}

maketab <- function(tags, bib, position = "left", title = FALSE) {
  
  tags_table <- extract_tags(bib = bib)
  
  all_tags <- unique(tags_table$tag)
  
  if (!all(tags %in% all_tags)) stop("Some tags are not found in any reference")
  
  publis <- tags_table$name[tags_table$tag %in% tags]
  
  selected <- bib[publis]
  
  tab <- tibble::tibble(Title = selected$title, Key = names(selected))
  
  tab$Title <- stringr::str_replace_all(tab$Title, "\\{|\\}|\\\\textit", "") ## remove LaTeX code created by RefManageR
  tab$Title <- stringr::str_replace_all(tab$Title, "[[:blank:]]\\)", "\\)") ## remove extra spaces before )
  tab$Key <- paste0("@", tab$Key)
  tab$pdf <- "[<img src='images/pdf.png' width=20px style='float:none; vertical-align:middle; padding:0 0 0 0'/>]"
  tab$pdf <- paste0(tab$pdf, "(pdfs/", publis, ".pdf)")
  
  ## sorting
  tab$year <- as.numeric(stringr::str_extract(tab$Key, pattern = "[[:digit:]]{4}"))
  tab <- tab[order(-tab$year, tab$Key), ]
  tab$year <- NULL
  
  if (!title) {
    tab <- tab[, c("Key", "pdf")] ## Remove Title for now
    tab <- kableExtra::kable(tab, col.names = c("", ""), escape = FALSE,  align = "rc")
    tab <- kableExtra::column_spec(tab, column = 2, width = "20px")
  } else {
    tab$pdf <- stringr::str_replace_all(tab$pdf, "20px", "100px")
    tab <- kableExtra::kable(tab, col.names = c("", "", ""), escape = FALSE,  align = "lrc") 
    tab <- kableExtra::column_spec(tab, column = 1, extra_css = "line-height: 1.3")
    tab <- kableExtra::column_spec(tab, column = 2, width = "200px")
    tab <- kableExtra::column_spec(tab, column = 3, width = "20px")
  }
  
  tab <- kableExtra::kable_styling(tab, position = position)
  tab
}
```

Publications are sorted by multiple criteria and you can jump directly to the following categories: [most recent](#recent), [selection](#best), [taxonomic groups](#taxon), [themes](#themes).

**Hover on top of a reference to reveal its details** (or click if using a touch screen).

Only publications created while being part of the Data Zoo Gang are listed here.
For the full list of publications of a given team member, please refer to: 

- Google Scholar:
[Alexandre Courtiol](https://scholar.google.com/citations?user=tyq-eagAAAAJ),
[Liam D. Bailey](https://scholar.google.com/citations?user=ys7AYRUAAAAJ),
[Ian J. Rickard](https://scholar.google.com/citations?user=eTP-OeMAAAAJ).

- ORCID:
[Alexandre Courtiol](https://orcid.org/0000-0003-0637-2959), 
[Liam D. Bailey](https://orcid.org/0000-0002-8226-9454),
[Colin Vullioud](https://orcid.org/0000-0001-9218-1571),
[Ian J. Rickard](https://orcid.org/0000-0003-0762-1689),
[Camille Martinez-Almoyna](https://orcid.org/0000-0002-3624-7932).

## Most recent {#recent}

Our publications from the last 2 years are:

```{r, echo=FALSE}
maketab(tags = "recent", bib = bib, position = "center", title = TRUE)
```

## Selection {#best}

Here is a short selection of recent DZG publications we are particularly proud of and which best reflect our involvement and mindset.

- @rickard_mothers_2022 [<img src="images/pdf.png" width=20px style="float:none; vertical-align:middle; padding:0 0 0 0"/>](pdfs/rickard_mothers_2022.pdf) is paper investigating the relationship between twinning and fertility in Humans and illustrating a new type of Simpson's paradox -- published in *Nature Communications*.
- @bailey_bird_2022 [<img src="images/pdf.png" width=20px style="float:none; vertical-align:middle; padding:0 0 0 0"/>](pdfs/bailey_bird_2022.pdf) is a meta-analysis of 67 populations of great and blue tits investigating the link between the phenotypic plasticity on their phenology and climate change -- published in *Nature Communications*.
- @culina_connecting_2021 [<img src="images/pdf.png" width=20px style="float:none; vertical-align:middle; padding:0 0 0 0"/>](pdfs/culina_connecting_2021.pdf) is a paper announcing the creation of the SPI-Birds network and its database for long-term population studies on birds -- published in *Journal of Animal Ecology*.
- @radchuk_adaptive_2019 [<img src="images/pdf.png" width=20px style="float:none; vertical-align:middle; padding:0 0 0 0"/>](pdfs/radchuk_adaptive_2019.pdf) is a meta-analysis quantifying precisely the extent to which animals (mostly birds) are adapting to changing climate -- published in *Nature Communications*.
- @vullioud_social_2019 [<img src="images/pdf.png" width=20px style="float:none; vertical-align:middle; padding:0 0 0 0"/>](pdfs/vullioud_social_2019.pdf) is an in-depth study of social hierarchy in spotted hyenas revealing how social rank emerges from simple behavioural rules -- published in *Nature Ecology & Evolution*.
- @corbett_transition_2018 [<img src="images/pdf.png" width=20px style="float:none; vertical-align:middle; padding:0 0 0 0"/>](pdfs/corbett_transition_2018.pdf) is a review of the causes and consequences of demographic transitions in humans, with an emphasis on human health -- published in *Nature Reviews Genetics*.
- @lahdenpera_differences_2018 [<img src="images/pdf.png" width=20px style="float:none; vertical-align:middle; padding:0 0 0 0"/>](pdfs/lahdenpera_differences_2018.pdf) is a detailed study of how capture influences survival in Asian elephants -- published in *Nature Communications*.


## Taxonomic groups {#taxon}

We mostly work on birds and mammals (including humans) but sometimes venture to study other lifeforms, as well as viruses.

### Arthropods {#taxon_arthropod}

:::float-right
<img src="images/droso.jpg" width="200px">
```{r, echo=FALSE}
maketab(tags = "taxon_arthropod", bib = bib, position = "left")
```
<!-- @debelle_mate_2017 [<img src="images/pdf.png" width=20px style="float:none; vertical-align:middle; padding:0 0 0 0"/>](pdfs/debelle_mate_2017.pdf) -->
:::

### Bats {#taxon_bat}

:::float-left
<img src="images/bat_costarica.jpg" width="200px">
```{r, echo=FALSE}
maketab(tags = "taxon_bat", bib = bib, position = "right")
```
:::

### Birds {#taxon_bird}

:::float-right
<img src="images/flamingo.jpg" width="200px">
```{r, echo=FALSE}
maketab(tags = "taxon_bird", bib = bib, position = "left")
```
:::

### Carnivores {#taxon_carnivore}
:::float-left
<img src="images/hyena_portrait.jpg" width="200px">
```{r, echo=FALSE}
maketab(tags = "taxon_carnivore", bib = bib, position = "right")
```
:::

### Elephants {#taxon_elephant}

:::float-right
<img src="images/asian_elephant_working.jpg" width="200px">
```{r, echo=FALSE}
maketab(tags = "taxon_elephant", bib = bib, position = "left")
```
:::

### Marsupials {#taxon_marsupial}

```{r, echo=FALSE}
maketab(tags = "taxon_marsupial", bib = bib, position = "right")
```

### Plants {#taxon_plant}

:::float-right
<img src="images/leucadendron_rubrum.jpg" width="200px">
```{r, echo=FALSE}
maketab(tags = "taxon_plant", bib = bib, position = "left")
```
:::

### Primates (including humans) {#taxon_primate}
:::float-left
<img src="images/baboon_tsaobis.jpg" width="200px">
```{r, echo=FALSE}
maketab(tags = "taxon_primate", bib = bib, position = "right")
```
:::

### Rodents {#taxon_rodent}

:::float-right
<img src="images/porcupine.jpg" width="200px">
```{r, echo=FALSE}
maketab(tags = "taxon_rodent", bib = bib, position = "left")
```
:::

### Ungulates {#taxon_ungulate}
:::float-left
<img src="images/zebra_southafrica.jpg" width="200px">
```{r, echo=FALSE}
maketab(tags = "taxon_ungulate", bib = bib, position = "right")
```
:::

### Viruses {#taxon_virus}

```{r, echo=FALSE}
maketab(tags = "taxon_virus", bib = bib, position = "left")
```

### All & none (purely theoretical) {#taxon_none}

```{r, echo=FALSE}
maketab(tags = "taxon_none", bib = bib, position = "right")
```


## Themes {#themes}

We study [wildlife conservation](#conservation), the ecology and evolution of [life history traits](#traits), [animal behaviour](#behaviour), [morphology](#morphology), as well as various aspects of [health](#health) in wildlife populations.
We also design [methods](#methods) that can be applied to study these themes.

### Wildlife conservation {#conservation}

#### Climate change {#conservation_climate}

```{r, echo=FALSE}
maketab(tags = "conservation_climate", bib = bib, position = "left", title = TRUE)
```

#### Habitat degradation (including urbanisation) {#conservation_degradation}

```{r, echo=FALSE}
maketab(tags = "conservation_degradation", bib = bib, position = "left", title = TRUE)
```

#### Reproduction and management {#conservation_reproduction}

```{r, echo=FALSE}
maketab(tags = "conservation_reproduction", bib = bib, position = "left", title = TRUE)
```

#### Effect of capture and captivity {#conservation_captivity}

```{r, echo=FALSE}
maketab(tags = "conservation_captivity", bib = bib, position = "left", title = TRUE)
```


### Life history {#traits}

#### Lifespan {#traits_lifespan}

```{r, echo=FALSE}
maketab(tags = "traits_lifespan", bib = bib, position = "left", title = TRUE)
```

#### Mating, reproductive success and fitness {#traits_fitness}

```{r, echo=FALSE}
maketab(tags = "traits_fitness", bib = bib, position = "left", title = TRUE)
```

#### Effect of climate and seasonality {#traits_climate}

```{r, echo=FALSE}
maketab(tags = "traits_climate", bib = bib, position = "left", title = TRUE)
```

#### Effect of demography {#traits_demography}

```{r, echo=FALSE}
maketab(tags = "traits_demography", bib = bib, position = "left", title = TRUE)
```


### Behaviour {#behaviour}

#### Perception {#behaviour_perception}

```{r, echo=FALSE}
maketab(tags = "behaviour_perception", bib = bib, position = "left", title = TRUE)
```

#### Mate choice {#behaviour_matechoice}

```{r, echo=FALSE}
maketab(tags = "behaviour_matechoice", bib = bib, position = "left", title = TRUE)
```

#### Migration and dispersion {#behaviour_migration}

```{r, echo=FALSE}
maketab(tags = "behaviour_migration", bib = bib, position = "left", title = TRUE)
```

#### Social hierarchy {#behaviour_hierarchy}

```{r, echo=FALSE}
maketab(tags = "behaviour_hierarchy", bib = bib, position = "left", title = TRUE)
```


### Health {#health}

#### Diseases {#health_diseases}

```{r, echo=FALSE}
maketab(tags = "health_diseases", bib = bib, position = "left", title = TRUE)
```

#### Immunity, stress and microbiome {#health_immunity}

```{r, echo=FALSE}
maketab(tags = "health_immunity", bib = bib, position = "left", title = TRUE)
```

#### MHC {#health_MHC}

```{r, echo=FALSE}
maketab(tags = "health_MHC", bib = bib, position = "left", title = TRUE)
```


### Morphology {#morphology}

#### Effect of climate {#morphology_climate}

```{r, echo=FALSE}
maketab(tags = "morphology_climate", bib = bib, position = "left", title = TRUE)
```

#### Effect of diet and lifestyle {#morphology_diet}

```{r, echo=FALSE}
maketab(tags = "morphology_diet", bib = bib, position = "left", title = TRUE)
```


### Methodological contributions {#methods}

#### Databases {#methods_databases}

```{r, echo=FALSE}
maketab(tags = "methods_databases", bib = bib, position = "left", title = TRUE)
```

#### R packages {#methods_packages}

```{r, echo=FALSE}
maketab(tags = "methods_packages", bib = bib, position = "left", title = TRUE)
```

#### Quantitative methods {#methods_quantitative}

```{r, echo=FALSE}
maketab(tags = "methods_quantitative", bib = bib, position = "left", title = TRUE)
```

#### Non-invasive methods {#methods_noninvasive}

```{r, echo=FALSE}
maketab(tags = "methods_noninvasive", bib = bib, position = "left", title = TRUE)
```

#### Wet lab and genomics {#methods_genomics}

```{r, echo=FALSE}
maketab(tags = "methods_genomics", bib = bib, position = "left", title = TRUE)
```
